### PR TITLE
Parameterize behaviors

### PR DESCRIPTION
**Pull request by vzvzv**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -123,6 +141,73 @@ will recursively (including subdirectories) load all the YAML files. For example
...
+    fruit: apple
+
+- key: tuesday-fruit
+  kind: Behavior
+  extend: fruit-of-the-day
+  values:
+    day: tuesday
+
+```
+
```
Comment (zhouzhuojie) [inline README.md]: 💯 

Can I ask for one more example that uses AbstractBehavior and Template together?

File: README.md [hunk 2]
```diff
@@ -123,6 +141,73 @@ will recursively (including subdirectories) load all the YAML files. For example
...
+          </human>
+```
+
+### Abstract Behaviors
+Abstract Behaviors can be used to parameterize some data
+
+```yaml
+- key: fruit-of-the-day
+  kind: AbstractBehavior
+  values:
```
Comment (sesquipedalian-dev) [inline README.md]: does `day: whatever` need to be included in this example?

Comment (vzvzv) [reply README.md]: no - this makes `fruit` essentially an optional parameter, while `day` is required

Comment (sesquipedalian-dev) [reply README.md]: got it

File: README.md [hunk 3]
```diff
@@ -123,6 +141,73 @@ will recursively (including subdirectories) load all the YAML files. For example
...
+    fruit: potato
```
Comment (gusk) [inline README.md]: 🥔

**Pull request merged**